pandas/core/frame.py

```diff
@@ -10570,11 +10570,11 @@ def quantile(
         --------
         >>> df = pd.DataFrame(np.array([[1, 1], [2, 10], [3, 100], [4, 100]]),
         ...                   columns=['a', 'b'])
-        >>> df.quantile(.1)
+        >>> df.quantile(.1, numeric_only=True)
```

Comment:
these should't be necessary, only add the option where it actually matters

Reply:
Done